packages/huma-widget/src/components/CreditLine/borrow/index.tsx

```diff
@@ -25,21 +24,23 @@ import { Success } from './6-Success'
 
 export interface CreditLineBorrowProps {
   poolName: `${POOL_NAME}`
+  poolType: `${POOL_TYPE}`
```

Comment:
```suggestion
  poolName: keyof typeof POOL_NAME;
  poolType: keyof typeof POOL_TYPE;
```

This might be cleaner, and you don't need to worry about casting them later.

Reply:
Updated